wbooks/src/redux/book/actions.ts

```diff
@@ -0,0 +1,11 @@
+export const BOOK_ACTION_NAMES = {
+  getBooks: 'getBooks',
```

Comment:
And the reducer identificator?

`getBooks: 'getBooks',` => `GET_BOOKS: '@@BOOK/GET_BOOKS'`

Reply:
Done!